aiobungie/internal/factory.py

```diff
@@ -387,7 +387,7 @@ def _set_character_attrs(self, payload: typedefs.JSONObject) -> character.Charac
             gender=enums.Gender(payload["genderType"]),
             race=enums.Race(payload["raceType"]),
             class_type=enums.Class(payload["classType"]),
-            emblem=assets.Image(str(payload["emblemBackgroundPath"])),
+            emblem=assets.Image(str(payload.get("emblemBackgroundPath"))),
             emblem_icon=assets.Image(str(payload["emblemPath"])),
```

Comment:
Can we also change the `emblem_icon` field.
```suggestion
            emblem_icon=assets.Image(payload.get("emblemPath")),
```